.changeset/fast-bulldogs-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'use-portal': patch
```

Comment:
```suggestion
'use-portal': minor
```

Removing the default export is a breaking change. However, this package is still in `v0` releases.